config/serverless.oblt.yml

```diff
@@ -110,4 +110,4 @@ xpack.ml.compatibleModuleType: 'observability'
 console.ui.embeddedEnabled: false
 
 # Disable role management (custom roles)
-xpack.security.roleManagementEnabled: false
+xpack.security.roleManagementEnabled: true
```

Comment:
Nit: as this defaults to true, I think we can just remove it, along with the comment.